editor/editor_node.cpp

```diff
@@ -4929,7 +4929,7 @@ bool EditorNode::is_run_playing() const {
 String EditorNode::get_run_playing_scene() const {
 	String run_filename = editor_run.get_running_scene();
 	if (run_filename.is_empty() && is_run_playing()) {
-		run_filename = GLOBAL_DEF("application/run/main_scene", ""); // Must be the main scene then.
+		run_filename = GLOBAL_DEF_BASIC("application/run/main_scene", ""); // Must be the main scene then.
```

Comment:
Settings shouldn't be defined twice. And I feel EditorNode is not the right place to define this, it's used and should be defined in Main IMO.